lib/util/http/index.ts

```diff
@@ -47,6 +47,10 @@ export interface HttpResponse<T = string> {
   authorization?: boolean;
 }
 
+export interface HttpError extends Error {
+  statusCode: number;
+}
```

Comment:
Use `HttpError` from `lib/util/types.ts`
```suggestion
```

See other comments for why not use `statusCode` 😉